ui/pages/settings/experimental-tab/experimental-tab.constant.js

```diff
@@ -0,0 +1,4 @@
+export const THEME_TYPE = {
+  DEFAULT: 'default',
```

Comment:
Could `DEFAULT` be changed to `LIGHT: 'light'` for convention ?

Reply:
As the light colors are default and are assigned to the `root:` without any `data-theme` attribute. The `data-theme="light"` won't have any css declarations assigned to it. Do you think this matters? Won't cause confusion?

Reply:
We went with `default` because there could come a time that `light` *isn't* the default

Reply:
Very true! I think default works here